rocksdb_admin/admin_handler.cpp

```diff
@@ -2182,4 +2183,18 @@ std::vector<std::string> AdminHandler::getAllDBNames() {
     return db_manager_->getAllDBNames();
 }
 
+void AdminHandler::initS3Tmp() {
+  auto s3_path = folly::stringPrintf("%ss3_tmp/", FLAGS_rocksdb_dir.c_str());
```

Comment:
This `s3_path` will also be used elsewhere right? Shall we construct the path for once and use it throughout the program, would also be less error-prone